src/man/sssd-ad.5.xml

```diff
@@ -363,6 +363,14 @@ DOM:dom1:(memberOf:1.2.840.113556.1.4.1941:=cn=nestedgroup,ou=groups,dc=example,
                             settings please refer to the
                             <quote>ad_gpo_map</quote> options.
                         </para>
+                        <para>
+                            Please note that current version of SSSD does
+                            not support Active Directory's built-in groups.
+                            Built-in groups (such as Administrators) in GPO
```

Comment:
Maybe also adding the related SID (S-1-5-32-544) would make it even less ambiguous?

Reply:
I added the SID there.